components/public-api/gitpod/v1/user.proto

```diff
@@ -0,0 +1,212 @@
+syntax = "proto3";
+
+package gitpod.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
+
+service UserService {
+  // GetAuthenticatedUser allows to retrieve the current user.
+  rpc GetAuthenticatedUser(GetAuthenticatedUserRequest) returns (GetAuthenticatedUserResponse) {}
+}
+
+message GetAuthenticatedUserRequest {}
+message GetAuthenticatedUserResponse {
+  User user = 1;
+}
+
+message User {
```

Comment:
wow it looks a lot, do we really need everything?

cc @svenefftinge if you think can have a look